StripeFinancialConnections/StripeFinancialConnections/Source/Helpers/FinancialConnectionsFont.swift

```diff
@@ -26,6 +26,8 @@ struct FinancialConnectionsFont {
         case medium
         /// 24 size / 32 line height / 700 weight
         case large
+        /// 28 size / 36 line height / 600 weight
```

Comment:
is 600 weight expected for extra large?

Reply:
thank you for noticing, I double-checked and changed everything to 600